oio/conscience/checker/tcp.py

```diff
@@ -24,11 +24,14 @@ def configure(self):
             if k not in self.checker_conf:
                 raise exc.ConfigurationException(
                     'Missing field "%s" in configuration' % k)
-        self.addr = (self.checker_conf['host'], self.checker_conf['port'])
+        addrinfo = socket.getaddrinfo(
```

Comment:
Be carefull to pass the option to avoid a costly DNS resolution.
We currently only manage addresses so you can give the AI_NUMERICHOST option.